README.md

```diff
@@ -291,6 +298,10 @@ For installation follow [the same steps](#installation) as above.
 
 ### Available Mixer models
 
+[`gs://mixer_models/sam`]: https://console.cloud.google.com/storage/mixer_models/sam/
```

Comment:
Is this needed multiple times?
Usually we put it after the first paragraph using the reference.